README.md

```diff
@@ -26,13 +26,14 @@ const topper = new TopperWebSdk();
 
 Where a `config` object can be passed to the constructor:
 
-| Property       | Default Value | Values                             |
-| -------------- | ------------- | ---------------------------------- |
-| environment    | `production`  | `production`, `sandbox`            |
-| is_android_app | `false`       | `true`, `false`                    |
-| is_ios_app     | `false`       | `true`, `false`                    |
-| theme          | `dark`        | `dark`, `light`                    |
-| variant        | `new-tab`     | `new-tab`, `same-tab`, `iframe`    |
+| Property       | Default Value | Values                          |
+| -------------- | ------------- | ------------------------------- |
+| environment    | `production`  | `production`, `sandbox`         |
+| is_android_app | `false`       | `true`, `false`                 |
+| is_ios_app     | `false`       | `true`, `false`                 |
+| locale         | `en`          | `en`, `pt`                      |
```

Comment:
let's add also `pt-BR` and `en-US`.

Those are supported aswell and we should follow the same behaviour of the rest api.

http://www.lingoes.net/en/translator/langcode.htm